examples/leaflet_core_library.html

```diff
@@ -16,10 +16,10 @@
   </head>
   <body>
     <div id="map"></div>
-    
+
     <!-- include cartodb.js library -->
     <script src="http://cdn.leafletjs.com/leaflet-0.7.3/leaflet.js"></script>
```

Comment:
This is http, but https has an invalid certificate

Reply:
Yes, it is! I have changed it to CloudFlare CDN ;)